.github/workflows/bump.yml

```diff
@@ -11,6 +11,8 @@ on:
         required: true
       bump-command:
         type: string
+        # the bump should happen only in release-candidate branch
+        default: GITHUB_REF=refs/heads/release-candidate npx semantic-release --dry-run --no-ci
```

Comment:
how about having no default? we could set it to something like `echo Meep meep no bump command && exit 1`

Reply:
done